collectors/ciscoduo/collector.js

```diff
@@ -154,15 +154,24 @@ class CiscoduoCollector extends PawsCollector {
         else {
             // This condition works if next page getting null or undefined
             const untilMoment = moment();
-
-            const { nextUntilMoment, nextSinceMoment, nextPollInterval } = calcNextCollectionInterval('no-cap', untilMoment, this.pollInterval);
-
-            return {
-                stream: curState.stream,
-                mintime: nextSinceMoment.unix(),
-                poll_interval_sec: nextPollInterval
-            };
-
+            if (curState.mintime) {
```

Comment:
I guess `curState.mintime` is always set and this `if` doesn't make a lot of sense.